core/src/main/resources/jenkins/model/Jenkins/oops.jelly

```diff
@@ -40,7 +40,10 @@ THE SOFTWARE.
       </div>
       <j:if test="${app.shouldShowStackTrace()}">
         <p>${%checkJIRA} ${%vote} ${%pleaseReport} ${%stackTracePlease} ${%checkML}</p>
-        <h2>${%Stack trace}</h2>
+        <h2>
+            ${%Stack trace}
+            <l:copyButton text="${h.printThrowable(request2.getAttribute('jakarta.servlet.error.exception'))}"/>
+        </h2>
         <pre style="margin:2em; clear:both">${h.printThrowable(request2.getAttribute('jakarta.servlet.error.exception'))}</pre>
```

Comment:
should this be stored in a variable so its not calculated twice?

roughly (untested):
```suggestion
        <j:set var="stacktrace" value="${h.printThrowable(request2.getAttribute('jakarta.servlet.error.exception'))}" />
        <h2>
            ${%Stack trace}
            <l:copyButton text="${stacktrace}"/>
        </h2>
        <pre style="margin:2em; clear:both">${stacktrace}</pre>
```